.microcks/overlays.yaml

```diff
@@ -0,0 +1,204 @@
+    update:
+      examples:
+        new_booking:
+          summary: New Booking
+          value: |-
+            {
+              "id": "{{ uuid() > put(bookingId) }}",
+              "trip_id": "{{ request.body/trip_id }}",
+              "passenger_name": "{{ request.body/passenger_name }}",
+              "has_bicycle": {{ request.body/has_bicycle }},
+              "has_dog": {{ request.body/has_dog }},
+              "links": {
+                "self": "https://api.example.com/bookings/{{ bookingId }}"
+              }
+            }
+
+  - target: $.paths["/bookings/{bookingId}"].parameters[?(@.name=="bookingId")].example
+    remove: true
+    
+  - target: $.paths["/bookings/{bookingId}"].parameters[?(@.name=="bookingId")]
+    update:
+      examples:
+        booking_1725ff48-ab45-4bb5-9d02-88745177dedb:
+          value: '1725ff48-ab45-4bb5-9d02-88745177dedb'
+        booking_bfc5af2c-f477-43c4-8bdf-a00bdb939d65:
+          value: 'bfc5af2c-f477-43c4-8bdf-a00bdb939d65'
+
+  - target: $.paths["/bookings/{bookingId}"].get.responses["200"].content["application/json"]["example"]
+    remove: true
+
+  - target: $.paths["/bookings/{bookingId}"].get.responses["200"].content["application/json"]
+    update:
+      examples:
+        booking_1725ff48-ab45-4bb5-9d02-88745177dedb:
+          summary: John Doe
+          value:
+            id: 1725ff48-ab45-4bb5-9d02-88745177dedb
+            trip_id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+            passenger_name: John Doe
+            has_bicycle: true
+            has_dog: true
+            links:
+              self: https://api.example.com/bookings/1725ff48-ab45-4bb5-9d02-88745177dedb
+        booking_bfc5af2c-f477-43c4-8bdf-a00bdb939d65:
+          summary: Billy Bikeless
+          value:
+            id: bfc5af2c-f477-43c4-8bdf-a00bdb939d65
+            trip_id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+            passenger_name: Billy Bikeless
+            has_bicycle: false
+            has_dog: true
+            links:
+              self: https://api.example.com/bookings/1725ff48-ab45-4bb5-9d02-88745177dedb
+
+
+  - target: $.paths["/bookings/{bookingId}/payment"].parameters[?(@.name=="bookingId")].example
+    remove: true
+    
+  - target: $.paths["/bookings/{bookingId}/payment"].parameters[?(@.name=="bookingId")]
+    update:
+      examples:
+        Card:
+          value: '1725ff48-ab45-4bb5-9d02-88745177dedb'
+        Bank:
+          value: '1725ff48-ab45-4bb5-9d02-88745177dedb'
+
+  - target: $.webhooks["newBooking"]["post"]["requestBody"].content["application/json"]["example"]
+    remove: true
+
+  - target: $.webhooks["newBooking"]["post"]["requestBody"].content["application/json"]
+    update:
+      examples:
+        new_booking:
+          summary: New Booking
+          value:
+            id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+            trip_id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
```

Comment:
again some two ids should probably be changed with the example booking id and eg trip id?